collection-scripts/gather

```diff
@@ -19,7 +19,7 @@ resources+=(ns/default ns/openshift ns/kube-system)
 resources+=(storageclasses persistentvolumes volumeattachments)
 
 # Networking Resources
-resources+=(clusternetworks hostsubnets)
+resources+=(network.config.openshift.io network.operator.openshift.io clusternetworks hostsubnets)
```

Comment:
This is already collected,https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_must-gather/94/pull-ci-openshift-must-gather-master-e2e-aws/45/artifacts/e2e-aws/must-gather/cluster-scoped-resources/config.openshift.io/networks.yaml so we don't need this explicitly.

Reply:
https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_must-gather/94/pull-ci-openshift-must-gather-master-e2e-aws/45/artifacts/e2e-aws/must-gather/cluster-scoped-resources/network.openshift.io/

Reply:
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_must-gather/94/pull-ci-openshift-must-gather-master-e2e-aws/45/artifacts/e2e-aws/must-gather/cluster-scoped-resources/operator.openshift.io/networks/cluster.yaml

Reply:
So providing we can't know if a given network object will exist as that depends on the plugin, I suggest we remove the clusternetworks and hostsubnets line from reousrces, and we add logic  in gather_network_logs to gather objects per the plugin used.

Reply:
We can do that, however I would like to see what you mean by 'logic' (I think we want to meet these scripts VERY simple.

Today the philosophy we use, is to try and collect data, fail quickly and move on (it's better to ask not get what we request (explain why) , and move on).